plugins/modules/zos_blockinfile.py

```diff
@@ -279,6 +279,20 @@
     marker_begin: "Begin Ansible Block Insertion 2"
     marker_end: "End Ansible Block Insertion 2"
     block: "{{ CONTENT }}"
+
+- name: Add a block to a gds
+  zos_blockinfile:
+    src: SYS1.SOME.PARTITIONED(0)
+    insertafter: EOF
+    block: "{{ CONTENT }}"
+
+- name: Add a block to dataset and backup in a new generation of gds
+  zos_blockinfile:
+    src: SOME.PARTITIONED.TEST
```

Comment:
small comment, looks like this is called partitioned but is used as a sequential, better to chance the name to not confuse users.